src/sentry/static/sentry/app/components/forms/selectAsyncField.jsx

```diff
@@ -84,8 +89,19 @@ class SelectAsyncField extends SelectField {
   };
 
   onQuery = query => {
-    // Used by legacy integrations
-    return {autocomplete_query: query, autocomplete_field: this.props.name};
+    let {depends} = this.props;
```

Comment:
Should there be a `propType` for this?

Reply:
https://github.com/getsentry/sentry/pull/9136/commits/74d28a93e8a36bcd0026ce6390efee242fc514e4